### PR TITLE
specify api keys in pelias config

### PR DESCRIPTION
**Pull request by missinglink**
note: branched off https://github.com/pelias/acceptance-tests/pull/50 please merge that branch in and merge 50 to master before merging this.

this PR allows you to specify api keys in your ~/pelias.json config for use in the acceptance tests

```javascript
{
  "mapzen": {
    "api_key": {
      "pelias.stage.mapzen.com": "pelias-stage-xxxxxx"
    }
  },
```

### CODE REVIEW

**Comment by dianashk** [comment]
I commented in the other PR to please remove the changes made to the tests. This should only add the API key functionality, not "fix" tests. The changes to those tests were suspect and happened in a separate PR which was partially reverted.

File: test_cases/feedback_pass.json [hunk 1]
```diff
@@ -36,11 +36,11 @@
       "expected": {
         "properties": [
           {
-            "text": "31 세종대로 서울시청 신관동, Seoul",
+            "text": "31 세종대로 서울시청 본관동, Seoul",
```
Comment (dianashk) [inline test_cases/feedback_pass.json]: these changes don't belong in this PR

**Comment by orangejulius** [comment]
I think this isn't even needed anymore. If the tests can handle the rate limiting, then we don't need an API key. Right?

File: test_cases/feedback_pass.json (hunk 1 above)
Comment (missinglink) [inline test_cases/feedback_pass.json]: Agreed, it's because I branched off https://github.com/pelias/acceptance-tests/pull/50 by accident

**Comment by dianashk** [comment]
I'm ok with putting this on hold until it's necessary. Let's mark as [DO-NOT-MERGE] for now.
